### PR TITLE
Add page tabs and language picker

### PR DESCRIPTION
**Pull request by stplasim**

### CODE REVIEW

**Comment by stplasim** [comment]
Hi @mmalfertheiner. I am not sure if this is the best way to define the tab style on `ButtonCustom`. It would give us the advantage of being able to define a primary and secondary. Since there are two styles.

However I am open for suggestions

File: databrowser/src/components/languageSwitcher/LanguageFilter.vue [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        }"
+        class="border-transparent"
+        role="button"
+        tabindex="0"
+        @click="changeLanguage(language)"
+      >
+        <span class="block py-1 px-2 font-semibold">{{ language }}</span>
+      </div>
+    </div>
+  </div>
```
Comment (mmalfertheiner) [inline databrowser/src/components/languageSwitcher/LanguageFilter.vue:49]: This should be a shareable component. The pagination has the exact same style.

Maybe you can call it `ButtonPillGroup`?

https://getbootstrap.com/docs/4.0/components/button-group/

File: databrowser/src/pages/DatasetDetailPage.vue [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+    <TabGroup>
+      <TabList
+        class="flex relative py-7 space-x-4 border-t border-b border-gray-300"
+      >
+        <Tab>Table view</Tab>
+        <Tab>Detail view</Tab>
+        <Tab>Raw Data</Tab>
+        <div class="absolute right-0">
+          <LanguagePicker
+            :languages="['DE', 'IT', 'EN', 'NL', 'CS', 'PL', 'FR', 'RU']"
```
Comment (mmalfertheiner) [inline databrowser/src/pages/DatasetDetailPage.vue:12]: I would extract this into a configuration somewhere global.

**Review by mmalfertheiner — CHANGES_REQUESTED** [review]

**Comment by mmalfertheiner** [comment]
Hi @stplasim,

could you please keep the ButtonPillGroup clean and remove all the mobile dialog logic from the component. For example, the pagination should use the ButtonPillGroup as well, but it behaves different on mobile.

File: databrowser/src/pages/DatasetDetailPage.vue [hunk 3]
```diff
@@ -8,10 +8,18 @@
...
+          </ButtonPill>
+
           <ButtonPillGroup
             :data="suportedLanguages"
-            :default-selected="defaultLanguage"
-            :initial-selected="initialSelected"
+            :initial-selected="currentSelected"
+            class="hidden md:inline-flex"
             @selected-change="changeLanguage"
           />
```
Comment (mmalfertheiner) [inline databrowser/src/pages/DatasetDetailPage.vue]: I would wrap this into a `LanguagePicker` component, to reduce the complexity of the DatasetDetailPage.

**Review by mmalfertheiner — CHANGES_REQUESTED** [review]

**Pull request merged**